### PR TITLE
Adding 'full' version of conda build

### PR DESCRIPTION
**Pull request by pjh5**

### CODE REVIEW

File: docker/jenkins/common/install_nccl.sh [hunk 1]
```diff
@@ -19,6 +19,8 @@ elif [[ "$UBUNTU_VERSION" == 14.04 ]]; then
   NCCL_DEB='nvidia-machine-learning-repo-ubuntu1404_4.0-2_amd64.deb'
 fi
 
+apt update
+apt-get install wget
```
Comment (pietern) [inline docker/jenkins/common/install_nccl.sh]: curl is already installed, you can use `curl -LO <url>` to make it write the file.

File: docker/jenkins/ubuntu/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,8 @@ RUN bash ./install_ccache.sh && rm install_ccache.sh
 # Install Python
 ARG PYTHON_VERSION
 ADD ./install_python.sh install_python.sh
-RUN bash ./install_python.sh && rm install_python.sh
+RUN if [ -n "${PYTHON_VERSION}" ]; then bash ./install_python.sh; fi
+RUN rm install_python.sh
```
Comment (pietern) [inline docker/jenkins/ubuntu/Dockerfile]: This can go after #1953 is merged.

File: docker/jenkins/build.sh [hunk 3]
```diff
@@ -51,7 +24,9 @@ else
   exit 1
 fi
 
-PYTHON_VERSION="$(echo "${image}" | perl -n -e'/py(\d+(\.\d+)?)/ && print $1')"
+if [[ "$image" == *py* ]]; then
+  PYTHON_VERSION="$(echo "${image}" | perl -n -e'/py(\d+(\.\d+)?)/ && print $1')"
+fi
```
Comment (pietern) [inline docker/jenkins/build.sh]: This can go after #1953 is merged.

**Comment by pjh5** [comment]
Just trying to get leveldb working at this point

**Pull request merged**